### PR TITLE
feat: design array schema

### PR DESCRIPTION
**Pull request by marina-mosti**
- Allow `SchemaForm` to interpret and display schema elements in a line, based on the following 
- Added units

```js
const SCHEMA = [
  [ {}, {}, {} ], // 3 inputs in a row. Possibly using `display: flex` as wrapper
  {}, // 1 input in a row
  [ {}, {} ], // 2 inputs in a row
  [ {} ], // also 1 input in a row
]
```

TODO:

- Have to look at how Lookup with work with this

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [ ] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: src/SchemaForm.vue [hunk 1]
```diff
@@ -58,29 +67,20 @@ export default {
...
-
-      return arraySchema.map(field => ({
-        ...field,
-        uuid: getID(field.model)
-      }))
-    })
+    const { parsedSchema } = useParsedSchema(props)
 
     watch(parsedSchema,
       (schema, oldSchema) => {
```
Comment (shentao) [inline src/SchemaForm.vue]: Could we extract this callback too a named function that describes what it does?

File: src/features/ParsedSchema.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  const parsedSchema = computed(() => {
+    const arraySchema = Array.isArray(props.schema)
+      ? props.schema
+      : Object.keys(props.schema).map(model => ({
+        ...props.schema[model],
+        model
+      }))
+
+    for (const index in arraySchema) {
```
Comment (shentao) [inline src/features/ParsedSchema.js]: What about?
```
const normalizedSchema = arraySchema.map(
  field => Array.isArray(field) ? field : [ field ]
)
```

**Review by shentao — APPROVED** [review]
Looks good! One change suggestion though.

File: src/features/ParsedSchema.js (hunk 2 above)
Comment (marina-mosti) [reply src/features/ParsedSchema.js]: Good call, I like it

File: tests/unit/SchemaForm.spec.js [hunk 3]
```diff
@@ -36,6 +36,57 @@ describe('SchemaForm', () => {
     expect(wrapper.findAllComponents(FormText)).toHaveLength(2)
   })
 
+  describe('array schema', () => {
```
Comment (tzhelyazkova) [inline tests/unit/SchemaForm.spec.js:39]: the first two tests below seem to do the same thing, so perhaps having the second one only (nested arrays) is sufficient

**Review by tzhelyazkova — APPROVED** [review]

**Pull request merged**